dev/sg/sg_insights.go

```diff
@@ -0,0 +1,120 @@
+package main
+
+import (
+	"encoding/base64"
+	"strings"
+
+	"github.com/keegancsmith/sqlf"
+	"github.com/urfave/cli/v2"
+
+	"github.com/sourcegraph/log"
+
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/sgconf"
+	"github.com/sourcegraph/sourcegraph/dev/sg/internal/std"
+	"github.com/sourcegraph/sourcegraph/internal/database"
+	"github.com/sourcegraph/sourcegraph/internal/database/basestore"
+	connections "github.com/sourcegraph/sourcegraph/internal/database/connections/live"
+	"github.com/sourcegraph/sourcegraph/internal/database/postgresdsn"
+	"github.com/sourcegraph/sourcegraph/internal/observation"
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+)
+
+var (
+	insightsCommand = &cli.Command{
+		Name:     "insights",
+		Usage:    "Tools to interact with Code Insights data",
+		Category: CategoryUtil,
```

Comment:
```suggestion
		Category: CategoryDev,
```

(it's sorted under `// Common dev tasks` above, and it does seem like a dev task command)